src/pages/challenges/get-healthy.js

```diff
@@ -1,34 +1,57 @@
-import Image from 'next/image'
+import styles from '../../styles/Challenges.module.css'
 import HeaderCard from 'components/cards/HeaderCard'
 import ChallengeCategoryList from 'components/challenges/ChallengeCategoryList'
-import styles from '../../styles/Challenges.module.css'
 import DatePrinter from '../../components/layout/DatePrinter'
+import {
+	getTodayTasksByCategory,
+	getTodayScoreByCategory
+} from 'model/helpers/today-tasks'
+import { useState } from 'react'
+import { useUserContext } from 'context/UserContext.js'
+
+export default function GetHealthy({ tasks, progress }) {
+	const [data, setData] = useState({ tasks: tasks, progress: progress })
+	const userId = 1
+	const ctx = useUserContext()
 
-export default function GetHealthy({ tasks }) {
 	const completedHandler = async (taskId) => {
-		const response = await fetch('http//localhost:3000/api/update-status', {
+		const taskPoints = data.tasks.find(
+			(task) => task.id === Number(taskId)
+		).points
+
+		const response = await fetch('http://localhost:3000/api/update-status', {
```

Comment:
Sorry for spying on your PRs - I think you should be able to make your fetch requests from the frontend to the relative url `/api/blah`, which would be preferred because it'll work regardless of what host/post/domain the app is deployed at

Reply:
Thank you very much, Jamie. Indeed, we should change it to
const response = await fetch('/api/update-status', {..}')